Add documentation around microbial txome indices (#127)
AlexsLemonade/refinebio-docs: docs/main_text.md

The `download_url` field will allow you to download the index.

###### Microbial transcriptome indices

For an individual microbial species, there are often multiple genome assemblies available.
Multiple assemblies from a species often reflect that multiple strains from that species have been characterized or established as laboratory strains.

> To process microbial RNA-seq data in refine.bio, domain experts select a _single_ "blessed" genome assembly to be used to generate transcriptome index.

Reviewer: ```suggestion
We use a single genome assembly, selected by domain experts, when we generate the transcriptome index used for each microbial species.
```

The single genome assembly is typically from a common laboratory strain.

> The "blessed" strain transcriptome index is used to quantify _all_ RNA-seq data from that species, regardless of a sample's reported strain of origin.

Reviewer: ```suggestion
This strain's transcriptome index is used to quantify _all_ RNA-seq data from that species, regardless of a sample's reported strain of origin.
```
